updateQDKVersion.ps1

```diff
@@ -0,0 +1,61 @@
+$ver = $args[0]
```

Comment:
Generally, it's more robust in PowerShell to use a `param()` block to declare parameters for a script than to unpack `$args` manually. That allows for typechecking, automatic help, and tab completion. You can even use [`[CmdletBinding()]`](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_functions_cmdletbindingattribute?view=powershell-7) to make sure that parameters are mandatory, providing built-in error messages when they aren't present.
```suggestion
[CmdletBinding()]
param(
    [Parameter(Mandatory=$true)]
    [string] $Version
);
```